hw/ip/rv_core_ibex/doc/_index.md

```diff
@@ -86,18 +87,32 @@ The Ibex crash dump state includes the following information:
 * The current PC
 * The next PC
 * The last data access address
-* The last exception address
+* The last exception address (`mtval`)
```

Comment:
Could you somehow document the word indices of all 8 dump words?

Reply:
can you have a look at the latest and see if that's what you had in mind? really i should do a table..but i got lazy...